Makefile

```diff
@@ -100,7 +100,7 @@ clippy_core: install_rs_check_toolchain
 .PHONY: clippy_boolean # Run clippy lints enabling the boolean features
 clippy_boolean: install_rs_check_toolchain
 	RUSTFLAGS="$(RUSTFLAGS)" cargo "$(CARGO_RS_CHECK_TOOLCHAIN)" clippy \
-		--features=$(TARGET_ARCH_FEATURE),boolean \
+		--features=$(TARGET_ARCH_FEATURE),boolean,internal-keycache \
```

Comment:
are there internal-keycache-feature still needed after you moved the keycache thing out of shortint ?